### PR TITLE
Milestone #6

### PR DESCRIPTION
**Pull request by BregornOriginal**
# Create the views to the blog app

### In this milestone were added the next changes 

- The home page shows a list of users
- Each user has his corresponding page
- For each user page were added the list of post 
- For each post, you can have a better description of it
- Some methods implemented in models, like last comments, were used to show the last 5 comments
- Added some basic styles following the medium warframe

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1 +1,32 @@
...
+    </div>
+</section>
+
+<section>
+  <% @posts.each do |post| %>
+    <article class='post-card'>
+      <div>
+        <a href='posts/<%=post.id%>'><p><%= post.title %></a></p>
+        <p maxlength= 150><%= post.text %></p>
+      </div>
```
Comment (Davidosky007) [inline app/views/posts/index.html.erb:17]: * Kindly use `post.recent_comments`  method to render 5 most recent comments, you have this methods created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md). to render the user's 5 most recent posts. [as mentioned in the project requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.)

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1 +1,29 @@
...
+</article>
+
+<section>
+<% @posts.each do |post| %>
+  <article class='post-card'>
+    <div>
+      <p><%= post.title%></p>
+      <p><%= post.text%></p>
+    </div>
+  <div class='counters'>
```
Comment (Davidosky007) [inline app/views/users/show.html.erb:22]: * Kindly use `#recent_post` method to render the user's 3 most recent posts of a user, you have this methods created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md). to render the user's 3 most recent posts. [as mentioned in the project requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @BregornOriginal 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
* Good looking readme file 👍 
* No linter errors 👍 
* Proper use of Gitflow 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/index.html.erb (hunk 1 above)
Comment (BregornOriginal) [reply app/views/posts/index.html.erb:17]: Hi @Davidosky007 tanks a lot for your review, I added next to that's lines of code, a conditional to show the recent comments if the post has.

**Review by aimemalaika — APPROVED** [review]
Hi @BregornOriginal, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉

- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**